specification/containerservices/resource-manager/Microsoft.ContainerService/preview/2018-08-01-preview/managedClusters.json

```diff
@@ -949,11 +892,35 @@
         "osType": {
           "$ref": "#/definitions/OSType",
           "description": "OsType to be used to specify os type. Choose from Linux and Windows. Default to Linux."
+        },
+        "maxCount":{
+          "type": "integer",
```

Comment:
Please check these properties for `readonly` ness

Reply:
The value should be passed by customer, and default value is nil. so we are good here